web/js/codeworld_shared.js

```diff
@@ -225,31 +291,58 @@ function registerStandardHints(successFunc)
         }
 
         var found = [];
-
+        var hints = Object.keys(codeWorldSymbols)
```

Comment:
Did you mean `codeWorldBuiltinSymbols`?

It's an interesting idea to highlight symbols defined in the current module, as well.  But I'm not sure this accomplishes that, because you'd have to re-register the `hint` helper every time the code changes, and you don't do that, right?

Reply:
It works. Don't need to re-register helper, logic don't changes. Every time user call `hint` helper make new completions source (`data`) from `codeWorldSymbols`.
Probably, code from `var hints = Object.keys(codeWorldSymbols)` with getting defined symbols and sorting of found completions should run on edit, not on hint.
![image](https://user-images.githubusercontent.com/5370736/52208067-6b2ca380-2890-11e9-8b5b-beda194d6257.png)